docs/community/tutorials-contest/2022/local-subnet-development/README.md

```diff
@@ -0,0 +1,390 @@
+  2. Click on the account image
+  3. Click "Import Account"
+  4. Select type: Private Key
+  5. Enter private key of the default airdrop account, which is `56289e99c94b6912bfc12adc093c9b51124f0dc54ac7a766b2bc5ccf558d8027`
+  6. Click "Import"
+
+### Step 4.2: Using Hardhat
+
+To interact with the Subnet using Hardhat, refer to [Using Hardhat with the Avalanche C-Chain](https://docs.avax.network/dapps/smart-contracts/using-hardhat-with-the-avalanche-c-chain). It is very similar to interacting with C-Chain. You only have to change `hardhat.config.ts` file. Inside that file, find the exported js object and inside of it find `networks`. Add a new network which will be your Subnet.
+
+```typescript
+subnet: {
+  url: "<yourRpcUrl>",
+  chainId: <yourChainId>,
+  accounts: ["<accounts-private-key>"]
+}
+```
+
+Example Values:
+
+```typescript
+subnet: {
+  url: "http://127.0.0.1:37868/ext/bc/2ALrMJ74YHrq6gRXzZkmYaAx6tJhshybkWr8m71r56E7Cv25Qf/rpc",
+  chainId: 676767,
+  accounts: ["56289e99c94b6912bfc12adc093c9b51124f0dc54ac7a766b2bc5ccf558d8027"]
+}
+```
+
+Example Updated File: [hardhat.config.ts](./hardhat.config.ts.md)
+
+Now you can run any commands ran in the tutorial with `--network subnet` parameter
+
+Example command:
+
+```bash
+yarn deploy --network subnet
+```
+
+## Step 5: Interact with Precompiles (Optional)
+
+If you have followed the tutorial as it is, you do not need this part. Since, in this tutorial we did not add any precompiles to the Subnet. Therefore, this step is optional and helpful only if you are trying to extend your Subnet with precompiles.
+
+To checkout current precompiles provided by Ava Labs refer to [this](https://docs.avax.network/subnets/customize-a-subnet#precompiles). There are 3 precompiles shared by Ava Labs at the time this documentation is written.
+
+- [Contract Deployer Allow List](https://docs.avax.network/subnets/customize-a-subnet#restricting-smart-contract-deployers): restricts the addresses who can deploy contracts
+- [Transaction Allow List](https://docs.avax.network/subnets/customize-a-subnet#restricting-who-can-submit-transactions): restricts the addresses who can send transactions
+- [Native Minter](https://docs.avax.network/subnets/customize-a-subnet#minting-native-coins): allows given addresses to mint native token
+
+This tutorial will show how to interact with them using Remix.
+
+:::caution
+
+Before trying to interact with any of the precompiles make sure to add them while [Creating the Subnet](#step-2-create-the-subnet). You can not add them afterwards.
+
+:::
+
+#### General Steps to Interact with Precompiles
+
+1. While creating the Subnet you will be prompted `Advanced: Would you like to add a custom precompile to modify the EVM?:` answer `Yes` then choose the precompile you would like to add. Make sure to add your address as admin by following `Add admin` and entering your address. Do not forget that you can always get more details by selecting `More info` inside the command line.
```

Comment:
Please explain what address is for `and entering your address` and how to get that address.